sphinx/texinputs/sphinx.sty

```diff
@@ -855,13 +856,24 @@
   }
 \fi
 
-% Define literal-block environment
-\RequirePackage{newfloat}
-\DeclareFloatingEnvironment{literal-block}
+% for captions of literal blocks
+\newcounter{literalblock}
```

Comment:
perhaps I should rather use `sphinxliteralblock` as counter name here, in line with our recent informal guidelines.